core/00_primordials.js

```diff
@@ -36,6 +36,15 @@
 (() => {
   const primordials = {};
 
+  if (!Symbol.dispose) {
```

Comment:
I think we should have these in deno_core -- wasn't sure exactly where the right place to install them would be.

Reply:
Is there a particular reason for it since V8 doesn't support `using` syntax right now anyway?

If we are to do it, I'd prefer we did this:
https://github.com/denoland/deno/blob/cc05dc142d1c57afd10cb08d84f99a25bd49678c/runtime/js/99_main.js#L78-L79

Reply:
I see that you use it in `checking`. I'm somewhat wary of adding this symbol here, could you move it to `checkin` setup code?

Reply:
Yep, that works!